sky/cli.py

```diff
@@ -3997,25 +3997,6 @@ def jobs_dashboard(port: Optional[int]):
             click.echo('Exiting.')
 
 
-# TODO(zhwu): Backward compatibility for the old `sky spot launch` command.
-# It is now renamed to `sky jobs launch` and the old command is deprecated.
-# Remove in v0.8.0.
-@cli.group(cls=_NaturalOrderGroup)
```

Comment:
There's a few usages of `sky spot launch` in `backward_compatibility_tests.sh`, should we update that too?

Reply:
was already removed in #4173